server/models/migrations/next/migrate.sql

```diff
@@ -176,3 +178,14 @@ UPDATE account_reference_type AS art SET art.label = 'FORM.LABELS.COST_CENTER' W
  */
 ALTER TABLE `account_reference` MODIFY COLUMN `abbr` VARCHAR(200) NOT NULL;
 ALTER TABLE `account_reference` MODIFY COLUMN `description` VARCHAR(200) NOT NULL;
+
+/**
+* author: @jmcameron
+* date: 2021-09-24
+* description: Add lost stock report menu item
+*/
+INSERT IGNORE INTO `unit` VALUES
+  (300, 'Lost Stock Report', 'TREE.LOST_STOCK_REPORT', 'Report on stock lost during depot transfers', 282, '/reports/lost_stock_report');
+
+INSERT IGNORE INTO `report` (`report_key`, `title_key`) VALUES
+   ('lost_stock_report', 'LOST_STOCK_REPORT');
```

Comment:
This should be `TREE.LOST_STOCK_REPORT`.

Reply:
Fixed.